edx_proctoring/views.py

```diff
@@ -469,18 +469,29 @@ def get(self, request, course_id):
             allowed_enrollments_users
         ).values('user_id', 'status', 'modified')
 
+        # get all of the last verified onboarding attempts of these users
+        last_verified_attempt_dict = get_last_verified_onboarding_attempts_per_user(
+            allowed_enrollments_users,
+            onboarding_exam.backend,
+        )
+
         # select a verified, or most recent, exam attempt per user
         onboarding_attempts_per_user = self._get_relevant_attempt_per_user(onboarding_attempts)
 
         onboarding_data = []
         for user in allowed_enrollments_users:
             user_attempt = onboarding_attempts_per_user.get(user.id, {})
+            other_verified_attempt = last_verified_attempt_dict.get(user.id)
 
-            data = {}
-            data['username'] = user.username
-            data['status'] = (InstructorDashboardOnboardingAttemptStatus
-                              .get_onboarding_status_from_attempt_status(user_attempt.get('status')))
-            data['modified'] = user_attempt.get('modified')
+            data = {'username': user.username}
+
+            if not user_attempt and other_verified_attempt:
+                data['status'] = InstructorDashboardOnboardingAttemptStatus.other_course_approved
```

Comment:
We'll also have to add this status to [proctored_exam_onboarding_view.js](https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/static/proctoring/js/views/proctored_exam_onboarding_view.js) - I believe it will just render as `other_course_approved` without changing it, but it won't be included in the filters.

Reply:
Got it. I will update that file.